website/docs/quick-start/configure-cli.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Configure CLI
+sidebar_position: 4
+sidebar_label: Configure CLI
+---
+
+In the previous step, we've decided on the following:
+
+- Use a monorepo to configure and provision two Terraform components into three AWS accounts and two AWS regions
+- The filesystem layout for the infrastructure monorepo
+- To be able to use [Component Remote State](/core-concepts/components/remote-state), we put the `atmos.yaml` CLI config file
+  into `/usr/local/etc/atmos/atmos.yaml` folder and set the ENV var `ATMOS_BASE_PATH` to point to the absolute path of the root of the repo
+
+Next step is to configure `atmos.yaml`.
+
+`atmos.yaml` configuration file is used to control the behavior of the `atmos` CLI. The file supports many features that are configured in different
+sections of the `atmos.yaml` file. For the description of all the sections, refer to [CLI Configuration](/cli/configuration).
+
+For the purpuse of this Quck Start, below is the minimum configuration required for Atmos to work with Terraform and to
```

Comment:
This seems like an excessive amount of "minimum" configuration for a quick start. Could we have atmos instead ship with a built in configuration to make this file almost entirely optional? Maybe override 1-2 parameters, but not 60 lines.

Reply:
(obviously out of scope for this PR)